db/dcrsqlite/apisource.go

```diff
@@ -1328,7 +1337,20 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 		KnownTransactions: numberMaxOfTx,
 		KnownFundingTxns:  numReceiving,
 		KnownSpendingTxns: numSpending,
+		IsDummyAddress:    false,
+	}
+}
+
+// IsZeroHashP2PHKAddress checks if given address is dummy
+// see https://github.com/decred/dcrdata/issues/358 for details
+func IsZeroHashP2PHKAddress(checkAddressString string, params *chaincfg.Params) bool {
+	zeroed := [20]byte{}
+	address, err := dcrutil.NewAddressPubKeyHash(zeroed[:], params, 0)
+	if err != nil {
+		return false //invalid state, should panic
```

Comment:
So log the error, or return the error as well.  If `err != nil`, it's because `params` wasn't valid, or the length of a pubkey hash is no longer 20.  In any case, the error would be informative.